R/error_bars.R

```diff
@@ -38,29 +37,99 @@
 #' 
 #' gs <- gsplot() %>%
 #'    points(1:10, 1:10) %>%
-#'    error_bar(5, 5, y.high=1) 
+#'    error_bar(x=5, y=5, y.high=1, col="green") 
 #' gs 
 error_bar <- function(object, ...) {
   override("gsplot", "error_bar", object, ...)
 }
 
 
-error_bar.gsplot <- function(object, x, y, y.high=0, y.low=0, x.high=0, x.low=0, 
-                             epsilon=0.1, ..., legend.name=NULL, side=c(1,2)){
+error_bar.gsplot <- function(object, ..., legend.name=NULL, side=c(1,2)){
```

Comment:
I tried hard to get those defaults to work. It's not as simple as listing the y.high's and others along with the `...`, I think because of that lazy eval? I'll give it another go....maybe I missed something the first time I tried. Currently, if I just do this:

```
fun.name='error_bar'
  
  args <- list(x=x, y=y, y.high=y.high, y.low=y.low, x.high=x.high, x.low=x.low, epsilon=epsilon, ...)
    
  object <- gather_function_info(object, fun.name, args, legend.name=legend.name, side=side)
  
```

(instead of what's in the PR right now), I get the error:
```
Error in is.finite(side.vals) : 
  default method not implemented for type 'list'
```
coming back from `gather_function_info`